ports/cef/window.rs

```diff
@@ -104,29 +104,29 @@ impl Window {
         vec![WindowEvent::Idle]
     }
 
-    fn cursor_type_for_cursor(&self, cursor: Cursor) -> cef_cursor_type_t {
+    fn cursor_type_for_cursor(&self, cursor: CursorKind) -> cef_cursor_type_t {
         match cursor {
-            Cursor::None => return cef_cursor_type_t::CT_NONE,
-            Cursor::ContextMenu => return cef_cursor_type_t::CT_CONTEXTMENU,
-            Cursor::Grabbing => return cef_cursor_type_t::CT_GRABBING,
-            Cursor::Crosshair => return cef_cursor_type_t::CT_CROSS,
-            Cursor::Copy => return cef_cursor_type_t::CT_COPY,
-            Cursor::Alias => return cef_cursor_type_t::CT_ALIAS,
-            Cursor::Text => return cef_cursor_type_t::CT_IBEAM,
-            Cursor::Grab | Cursor::AllScroll =>
+            CursorKind::None => return cef_cursor_type_t::CT_NONE,
```

Comment:
Isn't it the same thing without the `return`'s? I think we can remove them and simplify the function just a bit.

Reply:
No, we don't need to use `return` if this pattern matching will just return the value!
IMO, it's more clear if we don't use `return` here!

Reply:
Yeah, no return needed :)